cloudant-sync-datastore-core/build.gradle

```diff
@@ -126,3 +126,54 @@ compileToolsJava {
     // tools.jar needed for custom doclet
     classpath += files(org.gradle.internal.jvm.Jvm.current().toolsJar)
 }
+
+//
+// Generate client.properties
+//
+
+//task for generating a client properties file
+class ClientProperties extends DefaultTask {
+
+    //allow this to be configured, default to client.properties
```

Comment:
This comment is not correct based on the default set here. I don't mind if the comment or the default changes, either works as the file is explicitly set anyway.